data/mods/PKs_Rebalance/pk_critter_doom.json

```diff
@@ -0,0 +1,508 @@
+    "description":"A towering monster vaguely reminicent of a satyr.  As a member of the ruling species of its domains, it is given administrative control.  Its plasma blasts are deadly.",
+    "flags":["SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "POISON", "BONES", "PUSH_MON", "BLEED"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_hell_baron",
+    "name": "hell baron",
+    "species":"ABERRATION",
+    "default_faction":"upper_doom",
+    "symbol":"K",
+    "color":"i_red",
+    "size":"LARGE",
+    "material":"flesh",
+    "diff":30,
+    "aggression":100,
+    "morale":150,
+    "speed":90,
+    "melee_skill":7,
+    "melee_dice":2,
+    "melee_dice_sides":14,
+    "melee_cut":5,
+    "dodge":2,
+    "armor_bash":12,
+    "armor_cut":12,
+    "armor_stab":12,
+    "armor_acid":3,
+    "armor_fire":9,
+    "vision_day":50,
+    "vision_night":7,
+    "hp":200,
+    "death_function":["NORMAL"],
+    "special_attacks":[["SMASH", 40], ["GRAB", 30], ["FLESH_GOLEM", 20]],
+    "description":"A towering monster vaguely reminicent of a satyr.  A ranking member of the ruling species of its domains, it is given command.  Its plasma blasts are deadly.",
+    "flags":["SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "DESTROYS", "POISON", "BONES", "PUSH_MON", "BLEED"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_mancubus",
+    "name": "mancubus",
+    "species":"ABERRATION",
+    "default_faction":"upper_doom",
+    "symbol":"M",
+    "color":"ired",
+    "size":"LARGE",
+    "material":"flesh",
+    "diff":25,
+    "aggression":100,
+    "morale":120,
+    "speed":70,
+    "melee_skill":3,
+    "melee_dice":2,
+    "melee_dice_sides":12,
+    "melee_cut":0,
+    "dodge":1,
+    "armor_bash":10,
+    "armor_cut":6,
+    "armor_stab":10,
+    "armor_fire":17,
+    "armor_acid":4,
+    "vision_day":40,
+    "vision_night":3,
+    "hp":250,
+    "special_attacks":[["SMASH", 30], ["FLAMETHROWER", 60]],
+    "description":"A corpulent, pig-faced monster; its arms have been forcably replaced with huge flame-cannons.",
+    "flags":["SEES", "HEARS", "STUMBLES", "WARM", "BASHES", "GROUP_BASH", "DESTROYS", "POISON", "BLEED", "NO_BREATHE", "BONES", "FAT", "PUSH_MON"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_cacodemon",
+    "name": "cacodemon",
+    "species":"ABERRATION",
+    "default_faction":"doom",
+    "symbol":"C",
+    "color":"i_red",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":20,
+    "aggression":70,
+    "morale":70,
+    "speed":80,
+    "melee_skill":6,
+    "melee_dice":2,
+    "melee_dice_sides":15,
+    "melee_cut":10,
+    "dodge":3,
+    "armor_bash":10,
+    "armor_cut":16,
+    "armor_stab":10,
+    "armor_acid":6,
+    "armor_fire":6,
+    "vision_day":40,
+    "vision_night":10,
+    "hp":140,
+    "death_function":["NORMAL"],
+    "special_attacks":[
+        {
+            "type" : "bite",
+            "cooldown" : 20,
+            "min_mul" : 0.8
```

Comment:
That bite attack is a waste of turn here.
It deals at most 5 damage - to an unarmored player.